scalecube-configuration-api/src/main/java/io/scalecube/configuration/api/ReadEntryHistoryResponse.java

```diff
@@ -0,0 +1,43 @@
+package io.scalecube.configuration.api;
+
+/**
+ * Represents a response to a key version history request.
+ */
+public class ReadEntryHistoryResponse {
```

Comment:
Why do we need such a response type?
Isn't enough to use `List<ReadEntryResponse>` as the return type of the service method?